### ISSUE TITLE
Visualize the training process for the model created using keras 

### ISSUE THREAD

**Issue by gshanbhag525** [CropAi/Data-Modeling#39]
Visualize the training process for the model created using keras .
Use matplotlib and seaborn libraries to display the training graph .
You can take inspiration from Tensorboard.

**Comment by xAdvitya** [comment]
I would like to work on this issue

**Comment by gshanbhag525** [comment]
@Advitya-sharma Any update on this issue??

**Comment by xAdvitya** [comment]
> @Advitya-sharma Any update on this issue??

I'm asking in slack chat but no one is responding , how I make the model running so that I can work on visualisation,please tell how to load images for running the code

**Comment by AdityaSingh17** [comment]
I would like to this issue up!

**Comment by gshanbhag525** [comment]
> > @Advitya-sharma Any update on this issue??
> 
> I'm asking in slack chat but no one is responding , how I make the model running so that I can work on visualisation,please tell how to load images for running the code
@Advitya-sharma
Here is the model https://github.com/CropAi/Data-Modeling/blob/master/Crop_AI_Keras.ipynb .....open it in colab....kaggle.json file is included in the repo. I hope this helps you. If you have any more doubts just drop a comment here instead of slack.

**Comment by gshanbhag525** [comment]
https://github.com/CropAi/Data-Modeling/blob/master/Upload%20Kaggle%20Dataset%20in%20colab.ipynb This notebook will give you more insight as how the dataset is imported into collab from kaggle. Also there is load_images python module file in the repo @Advitya-sharma

**Comment by xAdvitya** [comment]
> @Advitya-sharma Any update on this issue??

The code is ready, Should I make a separate .py file or include that in notebook ? and should I keep validation loss and validation accuracy in plot for future use ? as currently we are not calculating that

**Comment by gshanbhag525** [comment]
@Advitya-sharma Create a separate collab notebook and also include validation loss and validation accuracy in plot.
